includes/backends/class-omise-backend-fpx.php

```diff
@@ -18,6 +18,9 @@ public function initiate() {
 	 * @return array  of an available banks
 	 */
 	public function get_available_banks() {
+		if ( !$this->capabilities() ){
+			return null;
+		}
 		$providers = $this->capabilities()->getFPXBanks();
```

Comment:
Since `capabilities()` method calls Omise API, we can simply cache the value of the first call and avoid extra API call. Let's make same changes in other classes too.

```suggestion
                $capabilities = $this->capabilities();

		if ( !$capabilities ) {
			return null;
		}

		$providers = $capabilities->getFPXBanks();
```

Reply:
Updated here [1178a7b](https://github.com/omise/omise-woocommerce/pull/294/commits/1178a7b861be22f25afd997dd189b0b6e52fff67)